content/administration/odoo_sh/advanced/scheduled_actions.rst

```diff
@@ -0,0 +1,29 @@
+
+.. _odoosh-advanced-scheduled_actions:
+
+=================
+Scheduled actions
+=================
+
+Special worker
+--------------
+
+On Odoo.sh, cron jobs and incoming mails are handled by a specific
+worker that is not accounted in the workers on your subscription. It is a kind of "hidden free
+worker" bound to every Odoo.sh instance.
```

Comment:
- 100th character line break. see our [rst guidelines]([url](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character))

```suggestion
On Odoo.sh, cron jobs and incoming mails are handled by a specific worker that is not accounted in
the workers on your subscription. It is a kind of "hidden free worker" bound to every Odoo.sh
instance.
```